interface-definitions/load-balancing_reverse-proxy.xml.in

```diff
@@ -102,6 +102,39 @@
                   </leafNode>
                 </children>
               </node>
+              <node name="http-check">
+                <properties>
+                  <help>HTTP check configuration</help>
+                </properties>
+                <children>
+                  <leafNode name="method">
```

Comment:
This should be a multi node.

It also should have:

```
<completionHelp>
    <list>options get post put</list>
</completionHelp>
```

and the constraint should be changed to:

```
<constraint>
    <regex>(options|get|post|put)</regex>
</constraint> 
```

Reply:
Thanks for the inputs.
I will look into that and make another PR to improve that on current/cherry-pick here

Reply:
I've created #3338 to address the input.
But not sure why it should be a multi node? Only one method applies.

Reply:
If only one applies all is good then. It sounded like multiple methods are allowed at once.